### ISSUE TITLE
Remove DotNet and Kylix support

### ISSUE THREAD

**Issue by rlebeau** [IndySockets/Indy#131]
Remove DotNet and Kylix support

**Comment by darianmiller** [comment]
One artifact of this is that `IdNTLM.pas` contains invalid source

`type_3_message_header` is defined within  `{$IFDEF DOTNET}`


```pascal
function type_3_message_header.GetBytesLen: UInt32;
begin
  Result := inherited GetByteLen() + ? ;
end;
```

**Comment by rlebeau** [comment]
Indy hasn't been recompiled for DotNet in a very very long time. The code you mention would be eliminated as part of this cleanup.

**Comment by darianmiller** [comment]
Correct, I just wanted to document another reason for doing the ticket.
